package/iptables/iptables.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-IPTABLES_VERSION = 1.8.4
+IPTABLES_VERSION = 1.8.3
```

Comment:
Given that we are intentionally downgrading iptables does it make sense to leave a comment here as to why? Link to a known issue maybe?